app/routes/task_routes.py

```diff
@@ -0,0 +1,144 @@
+from flask import Blueprint, jsonify, make_response, request, abort
+from werkzeug.exceptions import RequestEntityTooLarge
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+import datetime
+import requests
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+task_bp = Blueprint("task", __name__, url_prefix="/tasks")
+
+# HELPER FUNCTION
+def valid_int(number, parameter_type):
+    try:
+        number = int(number)
+    except:
+        abort(400, {"error":f"{parameter_type} must be an int"})
+
+def get_task_from_id(task_id):
+    valid_int(task_id, "task_id")
+    return Task.query.get_or_404(task_id, description="{task not found}")
+
+def get_goal_from_id(goal_id):
+    valid_int(goal_id, "goal_id")
+    return Goal.query.get_or_404(goal_id)
```

Comment:
Minor note: Looks like you missed removing this helper function in your refactor.

```suggestion
```